src/openrct2/peep/Peep.h

```diff
@@ -736,6 +736,13 @@ struct Peep : rct_sprite_common
 struct Guest : Peep
 {
 public:
+    /**
+     * rct2: 0x699F5A
+     * al:thought_type
+     * ah:thought_arguments
+     * esi: peep
+     */
+    void InsertNewThought(PeepThoughtType thought_type, uint8_t thought_arguments);
```

Comment:
Ideally, this should be declared here, but not all instances of Peep are using the Guest struct yet. It should therefure be declared on the Peep struct, still.

The comment can be removed, too. It is only relevant for the implementation (cpp) file, not the header file.

Reply:
Yes sorry that was my mistake. I thought it was ready to go in Guest but no it must still be left inside of Peep.